automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xe
+
+# This script should be able to execute KubeMacPool
+# functional tests against Kubernetes cluster on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-workflow-k8s.sh
+
+teardown() {
+    make cluster-down
+}
+
+main() {
+    export KUBEVIRT_PROVIDER='k8s-1.17'
+
+    source automation/check-patch.setup.sh
+
+    cd ${TMP_PROJECT_PATH}
+    make cluster-down
+    make cluster-up
+    trap teardown EXIT SIGINT SIGTERM SIGSTOP
+    make cluster-operator-push
+    make cluster-operator-install
+
+    cd ${KUBEMACPOOL_TMP_PROJECT_PATH}
+    export KUBEVIRT_PROVIDER=external
+    export CNAO_PROVIDER=external
+    export KUBECONFIG=${TMP_PROJECT_PATH}/_kubevirtci/_ci-configs/${KUBEVIRT_PROVIDER}/.kubeconfig
```

Comment:
You can't depend on private generate directories here. Use https://github.com/kubevirt/cluster-network-addons-operator/blob/master/cluster/kubevirtci.sh#L34 instead